src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileFunction.java

```diff
@@ -291,6 +344,8 @@ private GetModuleFileResult getModuleFile(
         }
         result.moduleFile = moduleFile.get();
         result.registry = registry;
+        YankedVersionsUtil.getYankedInfo(registry, key, allowedYankedVersions, env.getListener())
```

Comment:
nit -- this reads a bit awkwardly. maybe `result.yankedInfo = YankedVersionsUtil.getYankedInfo(...).orElse(null);` (or just use an Optional in the struct).